assets/queries/terraform/alicloud/oss_bucket_allows_delete_from_all_principals/query.rego

```diff
@@ -0,0 +1,26 @@
+package Cx
+
+import data.generic.common as common_lib
+import data.generic.terraform as terra_lib
+
+CxPolicy[result] {
+
+	json_policy := input.document[i].resource.alicloud_oss_bucket[name].policy
+    
+    policy := common_lib.json_unmarshal(json_policy)
+	st := common_lib.get_statement(policy)
+	statement := st[_]
+	statement.Effect == "Allow"
+    terra_lib.anyPrincipal(statement)
+    action := statement.Action
+    contains(action[a],"DeleteBucket")
```

Comment:
I think `Action` can be set as string or as array. I suggest:
```suggestion
   	common_lib.containsOrInArrayContains(statement.Action, "deletebucket")
```

P.S. I set "deletebucket" because [common_lib.containsOrInArrayContains](https://github.com/Checkmarx/kics/blob/master/assets/libraries/common.rego#L100) uses lowercase in the action input.